AmplifyPlugins/API/Tests/AWSAPIPluginTests/Interceptor/AuthTokenURLRequestInterceptorTests.swift

```diff
@@ -31,6 +31,30 @@ class AuthTokenURLRequestInterceptorTests: XCTestCase {
         XCTAssertNotNil(headers[URLRequestConstants.Header.xAmzDate])
         XCTAssertNotNil(headers[URLRequestConstants.Header.userAgent])
     }
+    
+    func testAuthTokenInterceptor_ThrowsInvalid() async throws {
+        let mockTokenProvider = MockTokenProvider()
+        let interceptor = AuthTokenURLRequestInterceptor(authTokenProvider: mockTokenProvider,
+                                                         isTokenExpired: { _ in return true })
```

Comment:
added, it's part of AmplifyPlugins/API/Tests/AWSAPIPluginTests/Configuration/AWSAPIEndpointInterceptorsTests.swift since it's implemented in AWSAPIEndpointInterceptors